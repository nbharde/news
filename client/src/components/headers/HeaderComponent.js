import React from "react";

const HeaderComponent = (props) => <div className="heading" data-testid="heading_testId">News Articles</div>

export default HeaderComponent
