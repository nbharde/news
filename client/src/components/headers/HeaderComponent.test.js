import React from "react";
import { render } from "@testing-library/react";
import '@testing-library/jest-dom/extend-expect';

import HeaderComponent from "./HeaderComponent";

describe("Testing HeaderComponent", () => {
    it("renders a component", () => {
        const { queryByTestId } = render(<HeaderComponent />);
        expect(queryByTestId("heading_testId")).not.toBeNull();
    });

    it("matches snapshot", () => {
         const { container } = render(<HeaderComponent />);
        expect(container).toMatchSnapshot();
    });
});
