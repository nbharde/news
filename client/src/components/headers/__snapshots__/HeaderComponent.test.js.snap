// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HeaderComponent matches snapshot 1`] = `
<div>
  <div
    class="heading"
    data-testid="heading_testId"
  >
    News Articles
  </div>
</div>
`;
